src/internal/converters/eml/eml_test.go

```diff
@@ -162,6 +162,18 @@ func (suite *EMLUnitSuite) TestConvert_edge_cases() {
 				require.NoError(suite.T(), err, "setting attachment content")
 			},
 		},
+		{
+			name: "attachment without name",
```

Comment:
to gain extra confidence could we expand the tests here a little bit to cover:
* attachment with `GetName() == nil`
* multiple attachments with no name

Reply:
sure, I can do that.